src/game/shared/tf/tf_gamemovement.cpp

```diff
@@ -1237,7 +1243,7 @@ bool CTFGameMovement::CheckJumpButton()
 		return false;
 
 	// Cannot jump again until the jump button has been released.
-	if ( mv->m_nOldButtons & IN_JUMP )
+	if ( mv->m_nOldButtons & IN_JUMP && (!sv_autobunnyhopping.GetBool() || !bOnGround ) )
```

Comment:
Having !bOnGround in this statement means that this code will break parachutes. After this statement, there's an if statement that checks if scouts can air dash, which is completely skipped when this code is executed. Removing || !bOnGround fixes the issue.